test/cypress/integration/repo_management.js

```diff
@@ -38,6 +40,80 @@ describe('Repo Management tests', () => {
     cy.contains('Sync');
   });
 
+  it('retrieves and copies token from local repo list', () => {
+    cy.visit(localRepoUrl);
+    cy.get('[data-cy="get-token"]').contains('Get token').click();
+    cy.get('button').contains('Load token').click();
+    cy.get('[aria-label="Copyable input"]')
+      .invoke('val')
+      .then((token) => {
+        cy.get('[aria-label="Copy to clipboard"]').realClick();
+        cy.window().then((win) => {
+          win.navigator.clipboard.readText().then((copiedToken) => {
+            assert.equal(copiedToken, token);
```

Comment:
```suggestion
            expect(copiedToken).to.equal(token);
```

so we don't have to introduce another libary